moped-editor/src/views/projects/projectView/DeleteConfirmationModal.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import DeleteIcon from "@material-ui/icons/DeleteOutlined";
+
+const useStyles = makeStyles((theme) => ({
+  deleteButton: {
+    color: theme.palette.error.main,
+  },
+}));
+
+const DeleteConfirmationModal = ({
+  type,
+  submitDelete,
+  deleteConfirmationOpen,
+  setDeleteConfirmationOpen,
+  children
+}) => {
+  const classes = useStyles();
+
+  const handleDeleteClose = () => {
+    setDeleteConfirmationOpen(false);
+  };
+
+  return (
+    <span>
+      {children}
+      <Dialog
+        open={deleteConfirmationOpen}
+        onClose={handleDeleteClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {`Are you sure you want to delete this ${type}?`}
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button
+            variant="outlined"
+            className={classes.deleteButton}
```

Comment:
i believerthe MUI-preferred way to do this is to instead define the color within our theme so you can use the `color` prop. using the `color` prop will keep the hover/press/border styles consistent with other button components.

you can update our theme by copying [from here](https://github.com/cityofaustin/atd-moped/pull/824/files#diff-44937b1f7ff295c1e359f2a53ba72aa26f164bc292c836acb7e3cc0775934825) - which are the `error` colors i pulled from the MUI docs.